.github/workflows/premerge.yml

```diff
@@ -31,10 +31,8 @@ jobs:
       - run: pip install -r requirements-dev.txt
       - run: |
           export PYTHONPATH=$(pwd)
-          rm -rf /tmp/snapshot-storage
           cd tests/integration_test
           ./run_integration_tests.sh
-          rm -rf /tmp/snapshot-storage
```

Comment:
Should we create an unique folder for this test run so multiple tests can run simultaneously?

Reply:
Good suggestion.
This folder is either configured in project.yaml and use provision command for setup.

Or we do string substitute in fed server/client JSON file to replace those values configured in the poc folder